impeller/core/texture.h

```diff
@@ -20,11 +20,13 @@ class Texture {
 
   virtual void SetLabel(std::string_view label) = 0;
 
+  // Deprecated: use BlitPass::AddCopy instead.
```

Comment:
An issue to followup on cleaning this up?

Reply:
https://github.com/flutter/flutter/issues/147945